cli/users.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+class users():
```

Comment:
Use camelcase for the class name

Reply:
Feedback implemented in the latest commit.